kubernetes_tests/test_base.py

```diff
@@ -287,12 +292,12 @@ def start_job_in_kubernetes(self, dag_id, host):
         result_json = self.start_dag(dag_id=dag_id, host=host)
         dag_runs = result_json["dag_runs"]
         assert len(dag_runs) > 0
-        logical_date = None
+        run_after = None
         dag_run_id = None
         for dag_run in dag_runs:
             if dag_run["dag_id"] == dag_id:
-                logical_date = dag_run["logical_date"]
+                run_after = dag_run["run_after"]
                 dag_run_id = dag_run["dag_run_id"]
                 break
-        assert logical_date is not None, f"No logical_date can be found for the dag with {dag_id}"
-        return dag_run_id, logical_date
+        assert run_after is not None, f"No run_after can be found for the dag with {dag_id}"
+        return dag_run_id, run_after
```

Comment:
I don’t think this needs to change? Sicne we set logical_date in the request, logical date should not be None here. (We can probably change the error message).

Reply:
I changed it. I also added run_after for this check.